client/analytics/report/orders/table.js

```diff
@@ -156,7 +166,9 @@ class OrdersReportTable extends Component {
 				href: 'edit.php?s=' + coupon.code + '&post_type=shop_coupon',
 			} ) );
 
-			return [
+			const customRowData = applyFilters( ROWS_FILTER, false, row );
```

Comment:
That's a good point.  This could certainly happen if they're fetching data asynchronously.  I'll try and address this once we determine the best route for the above issue with filtering existing columns since this may change how things work slightly.